### PR TITLE
log-detective.com announced

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: _posts/2024-01-16-log-detective.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: AI-driven Build Failure diagnosis?
```
Comment (TomasTomecek) [inline _posts/2024-01-16-log-detective.md:2]: I'm not sure about the question mark at the end.

Can you please explain to me why the title looks like a question?

File: _posts/2024-01-16-log-detective.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+exciting new project at the intersection of artificial intelligence and error
+pattern detection in build logs. Unlike traditional approaches, *Log Detective*
+relies on your valuable input to kickstart its journey.
+
+What is Log Detective?
+----------------------
+
+Log Detective takes a new approach by seeking insights directly from
+maintainers willing to help other fellow maintainers.  Instead of using
+pre-trained generic models, we want to train AI only on sentences and language
```
Comment (TomasTomecek) [inline _posts/2024-01-16-log-detective.md:17]:
```suggestion
pre-trained generic models, we want to offer a customized AI system only on sentences and language
```

"train AI" seems unprofessional to me

**Review by TomasTomecek — APPROVED** [review]
Two nits, otherwise LGTM, well done!!

**Comment by praiskup** [comment]
Thanks! Fixed.

**Review by TomasTomecek — APPROVED** [review]
Let's go!!

**Pull request merged**